src/views/certifications/views/[slug]/components/heading-permalink/heading-permalink.module.css

```diff
@@ -0,0 +1,54 @@
+/** 
+* The permalink icon focus / hover styles should reflect 
+* the same behavior as the mdx-heading-permalink component
+* /src/components/dev-dot-content/mdx-components/mdx-heading-permalink
+**/
+
+.heading {
+	composes: hds-typography-display-400 from global;
+	composes: g-offset-scroll-margin-top from global;
+	font-weight: var(--token-typography-font-weight-bold);
+	color: var(--token-color-foreground-strong);
+	margin: 40px 0 32px 0;
+
+	&:hover {
```

Comment:
I referenced this behavior from the mdx permalinks component since that was Kelly's preference for the link icon!